dark/filter.py

```diff
@@ -116,6 +116,93 @@ def accept(self, title):
         return self.DEFAULT_ACCEPT
 
 
+class SequenceFilter:
+    """
+    Provide an acceptance test for sequences.
+
+    @param whitelist: If not C{None}, a C{set} of exact sequences that are always
+        acceptable.
+    @param blacklist: If not C{None}, a C{set} of exact sequences that are never
+        acceptable.
+    @param whitelistFile: If not C{None}, a C{str} filename containing lines
+        that give exact sequences that are always acceptable.
+    @param blacklistFile: If not C{None}, a C{str} filename containing lines
+        that give exact sequences that are never acceptable.
+    @param positiveRegex: If not C{None}, a C{str} regex that sequence sequences
```

Comment:
I don't understand what you mean by 'sequence sequences'

Reply:
Ah, this was a result of a string replacement. "sequence titles" got changed to "sequence sequences". I thought I had fixed them all. Fixed now, thanks!